src/jit/optcse.cpp

```diff
@@ -1574,17 +1574,16 @@ class CSE_Heuristic
 
         void InitializeCounts()
         {
+            m_Size = Expr()->GetCostSz(); // always the GetCostSz()
```

Comment:
I think I prefer the original version better. It's definitely not an error and I find the suggestion that it might be "un-optimized code" to be questionable. It's the job of the compiler to perform inlining and redundancy elimination.